### PR TITLE
Request Model issue #9

### PR DESCRIPTION
**Pull request by ar2rworld**
### What this PR does (required):
- Request model created

### CODE REVIEW

File: server/models/Request.js
Comment (ezeilo-su) [inline server/models/Request.js:4]: Use `cameCase` convention to name variables

File: server/models/Request.js
Comment (ezeilo-su) [inline server/models/Request.js:8]: Use `cameCase` convention to name variables

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
Nice work on implementing this model. Check the few comments below and address them so that your PR can be approved.
Happy coding! :tada:

File: server/models/Request.js
Comment (ugur-akin) [inline server/models/Request.js:29]: I find having only a single field for these (e.g. `status`) with an [enum validator](https://mongoosejs.com/docs/validation.html#built-in-validators) helps simplify the logic later, but I'll leave it up to you to implement this however you think is best 👍🏼

**Review by ugur-akin — APPROVED** [review]
Good and quick start 💨 💪🏼  Please address any pending comments before merging.

Just a general reminder about future PRs, please:
- Title your PRs in a self-explanatory way, closer to plain English (e.g. `Added a Mongoose Model for booking requests`)
- and [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) in your PR.

There is also a good [article](https://hugooodias.medium.com/the-anatomy-of-a-perfect-pull-request-567382bb6067) on this topic, feel free to give it a read if you're interested!

File: server/models/Request.js
Comment (ugur-akin) [inline server/models/Request.js:11]: This is fine for now, however, when we have a `User` model we would prefer to have this as a reference (see [Mongoose `populate`](https://mongoosejs.com/docs/populate.html). It might be a good idea to leave a `TODO` comment for these?

Comment (moffatethan) [reply server/models/Request.js:11]: Sitter ID will be referencing the Profile which is in progress, when that is megred in @ar2rworld I'll get you to refactor this file to reference it. No need to add in `TODO` comments 😄

**Pull request merged**
